### DISCUSSION TITLE
How to prevent small people from being generated in some squares?

### DISCUSSION THREAD

**Discussion by trihardseven** [Coyote-A/ultimate-upscale-for-automatic1111#42] [Q&A]
Using the recommended settings and 0.35 denoise my images almost always get small versions of the character being generated all over the background. Am I doing something wrong? Reducing denoise to around 0.15 fixes that but then its almost no different from normal upscaling.

**Comment by Jopezzia** [comment]
You can set larger tile size and remove person from promt. For now, this is all that can be done.
